.github/workflows/build.yml

```
@@ -37,7 +37,7 @@ jobs:
           java-version: ${{ matrix.java-version }}
 
       - name: Setup Gradle
-        uses: gradle/gradle-build-action@v2
+        uses: gradle/gradle-build-action@v3
```

```suggestion
        uses: gradle/actions/setup-gradle@v3
```

https://github.com/gradle/gradle-build-action/blob/main/README.md
https://github.com/gradle/actions/tree/main/setup-gradle